src/components/SignerPanel/SignerPanel.js

```diff
@@ -107,14 +107,16 @@ class SignerPanel extends React.PureComponent {
   // This is a temporary method to reshape the transaction bag
   // to the future format we expect from Aragon.js
   stateFromTransactionBag(bag) {
-    const { path, transaction } = bag
+    const { external, path, transaction } = bag
     const decoratedPaths = path.map(path => ({
       ...path,
       name: getAppName(this.props.apps, path.to),
     }))
 
     return {
       intent: (transaction && this.transactionIntent(bag)) || {},
+      external,
+      installed: !!getAppName(this.props.apps, transaction.to),
```

Comment:
```suggestion
      installed: !!this.props.apps.find(app => addressesEqual(app.proxyAddress, transaction.to))
```

`getAppName()` would be incorrect if the app was installed but didn't have a name (for whatever reason).